package.json

```diff
@@ -63,6 +63,9 @@
     "lint": "prettier -c . && lerna run --parallel workspace:lint --stream && npm run -s lint:markdown",
     "lint:fix": "prettier -w . && lerna run --parallel workspace:lint:fix --stream && npm run -s lint:markdown",
     "lint:markdown": "markdownlint-cli2 \"**/*.md\" \"#**/node_modules/**/*\" \"#SECURITY.md\"",
+    "loc:build": "cross-env-shell \"powershell -ExecutionPolicy Bypass -File ./Localize/build.ps1 && powershell -ExecutionPolicy Bypass -File ./Localize/copy-translations.ps1\"",
```

Comment:
Ah, nope, we aren't setting environment variables- I was just looking for a keyword that would let the script run on MacOS, Windows, and Linux and thought that cross-env-shell would work for that. How about  `pwsh -ExecutionPolicy Bypass -File ./Localize/build.ps1 && pwsh -ExecutionPolicy Bypass -File ./Localize/copy-translations.ps1`?

Reply:
I did this because the `pwsh `keyword doesn't work (it doesn't work on my Windows local box and also requires some overhead- I would need to download and install Powershell on Mac and Linux as well) and I wanted something that would allow the script to run cross-platform on Mac, Linux, and Windows. (Obviously `powershell `works only on Windows.) Is there another keyword I could use?

Reply:
This isn't going to work on Mac/Linux regardless, so IMO let's just remove cross-env-shell and call powershell directly.